src/main/resources/db/changelog/changeset/create-start-tables.sql

```diff
@@ -0,0 +1,50 @@
+-- liquibase formatted sql
+-- changeset yevhen-rozhylo:init-tables
+
+CREATE TABLE IF NOT EXISTS end_user(
```

Comment:
Please, use Liquibase instead of Flyway. You can read about advantages and disadvantages between Liquibase instead of Flyway.
Most of projects use Liquibase, so it's better for you to experience working with it.
Example, where you can read about how to use it: https://www.baeldung.com/liquibase-refactor-schema-of-java-app

Reply:
Oh, I didn't specify directly. 
Could you please make your changelog file not SQL formatted, but **XML formatted**, for example:
```
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<databaseChangeLog ...>
    
    <changeSet author="John" id="1439225004329-1">
        <createTable tableName="APP_USER">
            <column autoIncrement="true" name="id" type="BIGINT">
                <constraints primaryKey="true"/>
            </column>
            <column name="accessToken" type="VARCHAR(255)"/>
            <column name="needCaptcha" type="BIT(1)">
                <constraints nullable="false"/>
            </column>
            <column name="password" type="VARCHAR(255)"/>
            <column name="refreshToken" type="VARCHAR(255)"/>
            <column name="tokenExpiration" type="datetime"/>
            <column name="username" type="VARCHAR(255)">
                <constraints nullable="false"/>
            </column>
            <column name="preference_id" type="BIGINT"/>
            <column name="address" type="VARCHAR(255)"/>
        </createTable>
    </changeSet>
    ...
</databaseChangeLog>
```
Example to compare XML and SQL approaches: https://www.liquibase.org/get-started/best-practices

In projects, we usually use XML format Liquibase, where should be, for example, changelog-0.0.1.xml file, where all your changeSets will be located.

Reply:
Done:
* from now, liquibase is in .xml file where are all important change-sets of creation tables with columns, instead of using .yaml and .sql files